arrows/vtk/mesh_coloration.h

```diff
@@ -0,0 +1,162 @@
+/*ckwg +29
+ * Copyright 2016 by Kitware, SAS; Copyright 2017-2018 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither the name Kitware, Inc. nor the names of any contributors may be
+ *    used to endorse or promote products derived from this software without
+ *    specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef KWIVER_ARROWS_VTK_MESH_COLORATION_H_
+#define KWIVER_ARROWS_VTK_MESH_COLORATION_H_
+
+// KWIVER includes
+#include <vital/algo/video_input.h>
+#include <vital/config/config_block_types.h>
+#include <vital/types/camera_map.h>
+#include <vital/types/camera_perspective.h>
+
+// VTK Class
+class vtkFloatArray;
+class vtkPolyData;
+class vtkRenderWindow;
+
+#include <arrows/vtk/kwiver_algo_vtk_export.h>
+#include <vtkSmartPointer.h>
+
+#include <string>
+#include <vector>
+
+namespace kwiver {
+namespace arrows {
+namespace vtk {
+
+class KWIVER_ALGO_VTK_EXPORT mesh_coloration
+{
+
+public:
+  mesh_coloration();
+  mesh_coloration(kwiver::vital::config_block_sptr& videoConfig,
+                 std::string const& videoPath,
+                 kwiver::vital::config_block_sptr& maskConfig,
+                 std::string const& maskPath,
+                 kwiver::vital::camera_map_sptr& cameras);
+  virtual ~mesh_coloration()
+  {}
+
+  mesh_coloration(mesh_coloration const&) = delete;
+  mesh_coloration& operator=(mesh_coloration const&) = delete;
+
+  // Input mesh.
```

Comment:
You should add Doxygen style comments in the header to document this class and its member functions.  In KWIVER this is typically done in this style
```c++
/// Brief one-line description of function
/**
 * More detailed description of the function.  Often documenting parameters like this:
 *
 * \param  param_name  Description of this parameter
 */
```
Note that the brief comment needs three slashes `///` to be picked up by Doxygen